Gordon360/ApiControllers/HousingController.cs

```diff
@@ -61,6 +63,7 @@ public IHttpActionResult CheckIfHousingAdmin()
         /// <returns></returns>
         [HttpPost]
         [Route("admin/{id}")]
+        [StateYourBusiness(operation = Operation.ADD, resource = Resource.HOUSING)]
```

Comment:
Adding and removing a Hosuing Admin is not the same thing as adding and removing an application. This needs to be a separate resource. You should probably define a `HOUSING_ADMIN` resource similar to the `ADMIN` and `GROUP_ADMIN` resources.